predicators/envs/spot_env.py

```diff
@@ -2113,7 +2123,12 @@ def _detection_id_to_obj(self) -> Dict[ObjectDetectionID, Object]:
 
         for obj, pose in get_known_immovable_objects().items():
             detection_id = KnownStaticObjectDetectionID(obj.name, pose)
-            detection_id_to_obj[detection_id] = obj
+            if obj.name == "drafting_table":
```

Comment:
can you put this logic inside `get_known_immovable_objects` instead, so that other environments can use it?

Reply:
resolved in person